exercises/error-handling/error_handling_test.py

```diff
@@ -6,18 +6,26 @@
 class FileLike(object):
     def __init__(self):
         self.is_open = False
+        self.was_open = False
+        self.did_something = False
 
     def open(self):
-        self.is_open = True
-
-    def __enter__(self):
+        self.was_open = False
         self.is_open = True
 
     def close(self):
         self.is_open = False
+        self.was_open = True
+
+    def __enter__(self):
+        self.open()
 
     def __exit__(self):
```

Comment:
`__exit__` needs to accept 4 arguments, but we don't need them here, so we can just use `def __exit__(self, *args):`